docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -241,6 +241,18 @@ strapi templates:generate <path>
 
   Example: `strapi templates:generate ../strapi-template-name` will copy the required files and folders to a `template` directory inside `../strapi-template-name`
 
+## strapi content-types:generate-types
+
+Generate TypeScript typings for the project schemas.
+
+```sh
+strapi content-types:generate-types
+```
+
+:::note
+The `strapi content-types:generate-types` command only works in [TypeScript](/developer-docs/latest/development/typescript.md)-enabled projects.
```

Comment:
It also works for JavaScript projects, which will be nice since it means they'll also benefit from strong typing.
I think we can remove this block